src/System.Data.SqlClient/tests/FunctionalTests/Configurations.props

```diff
@@ -4,6 +4,8 @@
     <BuildConfigurations>
       netstandard-Unix;
       netstandard-Windows_NT;
+      netcoreapp-Unix;
```

Comment:
I made the netcoreapp configurations OS specific because otherwise if you just do "msbuild" it will pick the netstandard configurations, which is probably not what you want.

Reply:
I agree. @AfsanehR if you don't know of a reason why these tests could be OS specific, I will remove the OS suffixes.

Reply:
I think the reason why they are OS specific is because the AnyOS implementation assembly throws platform not supported exception.

Reply:
I think the reason why they are OS specific is because the AnyOS implementation assembly throws platform not supported exception.

Reply:
Ah OK